_docs/Components-Platform-Interface.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Components: Platform Communication Interface"
+---
+# The Platform Communication Interface
+*Formerly known as bridgehead component*
+
+## Abstract
+
+The platform communication interface is planned to be the interface between external modules and the Qabel communication layer. Therefor it shall provide communication channels to 
+
+* [Drop component](../Components-Drop/) and
+* [Box component](../Components-Box/)
+
+It is not targeted to be implemented in the Beta.
```

Comment:
You might be right regarding what we plan this interface to be. It might become an interface between modules and the platform in the future. However I would not state this but instead write what it actually is. We have this component right now as it is described shortly in my comment above: https://github.com/Qabel/qabel.github.io/pull/134/files#r51116957